nixos/modules/services/networking/keybase.nix

```diff
@@ -24,13 +24,24 @@ in {
 
   config = mkIf cfg.enable {
 
+    # Upstream: https://github.com/keybase/client/blob/master/packaging/linux/systemd/keybase.service
     systemd.user.services.keybase = {
       description = "Keybase service";
       unitConfig.ConditionUser = "!@system";
+      environment.KEYBASE_SERVICE_TYPE = "systemd";
+
+      script = ''
+        ${pkgs.keybase}/bin/keybase --use-default-log-file --debug service
```

Comment:
`--debug` seems unnecessary, but I guess since upstream does that it should be fine

Reply:
:heavy_check_mark: